dbld/builddeps

```diff
@@ -16,18 +16,18 @@ set -x
 # NOTE: If at all possible don't put anything here.
 function install_dbld_dependencies()
 {
-    case "${OS_PLATFORM}" in
-        centos-*)
+    case "${OS_DISTRIBUTION}" in
+        centos)
             $YUM_INSTALL wget yum-utils
             ;;
-        debian-*|ubuntu-*)
+        debian|ubuntu)
             apt-get update
             $APT_INSTALL wget gnupg2
-            if [ "${OS_PLATFORM}" = "ubuntu-xenial" ]; then
+            if [ "${IMAGE_PLATFORM}" = "ubuntu-xenial" ]; then
```

Comment:
This should be dependent on $RELEASE_CODE_NAME instead, although we currently only have one xenial based image.

Reply:
okey, I took your advice.